doc/api/http.md

```diff
@@ -652,7 +669,7 @@ added: v0.1.0
 * `response` {http.ServerResponse}
 
 Emitted each time there is a request. Note that there may be multiple requests
-per connection (in the case of keep-alive connections).
+per connection (in the case of HTTP Keep-Alive connections).
```

Comment:
You're saying this is the correct usage, right? Not that I should correct the usage here as well. I think this has the intended meaning.